bridges/YoutubeBridge.php

```diff
@@ -58,10 +58,44 @@ class YoutubeBridge extends BridgeAbstract {
 				'type' => 'number',
 				'title' => 'Maximum duration for the video in minutes',
 				'exampleValue' => 10
-			)
+			),
+			'attach_format' => array(
```

Comment:
I would like to have tested whether the `youtube-dl` command is installed before showing these options in the form but there are no computations allowed when initializing `const`s.
I thought I could simply move the parameters from the `PARAMETERS` to a method by overriding `getParameters()` from `BridgeAbstract` but it turns out `BridgeAbstract` uses `PARAMETERS` in a few locations directly instead of going via `getParameters()`. IMHO this should be refactored.

Reply:
Done